src/buildercore/bootstrap.py

```diff
@@ -19,6 +19,7 @@
 from boto.exception import BotoServerError
 from kids.cache import cache as cached
 from buildercore import cfngen
+import buildvars
```

Comment:
the problem to solve is to be able to read the node name/number (`journal--prod--1` or just `1` or `2`). Maybe we can get it in another way, like with the EC2 Tag? If so, the build vars do not need to be ported here and we will avoid this dependency